python_modules/libraries/dagster-polars/setup.py

```diff
@@ -45,7 +45,7 @@ def get_version() -> str:
         "deltalake": ["deltalake>=0.15.0"],
         "gcp": ["dagster-gcp>=0.19.5"],
         "test": [
-            "pytest>=7.3.1",
```

Comment:
is this supposed to be here

Reply:
it failed before i moved it here https://buildkite.com/dagster/dagster-dagster/builds/76501#018dd221-c4ba-4258-aefb-834405b9802e